core-api-library/library/src/main/java/net/gini/android/core/api/DocumentService.kt

```diff
@@ -0,0 +1,54 @@
+package net.gini.android.core.api
+
+import android.net.Uri
+import net.gini.android.core.api.models.Document
+import net.gini.android.core.api.response.PaymentRequestResponse
+import okhttp3.ResponseBody
+import retrofit2.Response
+import retrofit2.http.*
+
+interface DocumentService {
+
+    @POST("documents/")
+    suspend fun uploadDocument(@HeaderMap bearer: Map<String, String>, @Body bytes: ByteArray,  @Query("filename") fileName: String?, @Query("doctype") docType: String?): Response<Uri>
```

Comment:
The response for posting documents is empty. We need to get the document uri from the `location` response header:
![Screenshot 2022-08-26 at 17 52 45](https://user-images.githubusercontent.com/13149498/186944655-3e58d2fa-4133-4fda-8c32-10dccbf1eb15.png)